README.rst

```diff
@@ -112,8 +112,10 @@ Compatibility Notes
 +---------------------+----------------+
```

Comment:
Since we are dropping support for python 3.8, we need to add the constraint in the compatibility table for Quince < 8.0.0